test/python/circuit/library/test_adders.py

```diff
@@ -0,0 +1,88 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test adder circuits."""
+
+import unittest
+import numpy as np
+from ddt import ddt, data, unpack
+
+from qiskit.test.base import QiskitTestCase
+from qiskit.circuit import QuantumCircuit
+from qiskit.quantum_info import Statevector, partial_trace
+from qiskit.circuit.library import RippleCarryAdder, QFTAdder
+
+
+@ddt
+class TestAdder(QiskitTestCase):
+    """Test the adder circuits."""
+
+    def assertAdditionIsCorrect(self, num_state_qubits, adder, inplace, modular):
```

Comment:
Since most adders we have are not modular (and don't have the option to be) you could provide a default argument here if you want 
```suggestion
    def assertAdditionIsCorrect(self, num_state_qubits, adder, inplace, modular=False):
```

Reply:
Great suggestion, thank you! Pushed a commit with the change.